apps/swirl-docs/src/components/Icons/IconGridItem.tsx

```diff
@@ -75,22 +75,24 @@ const IconGridItem: FunctionComponent<IconGridProps> = ({
       </DesktopView>
       <MobileView>
         <div role={role}>
-          <a
-            id={`icon-popover-trigger`}
-            tabIndex={index === 0 ? 0 : -1}
-            ref={reference}
-            href={`#${icons[icon]?.name}`}
-            className="flex flex-col justify-center items-center py-4 border-1 rounded-lg"
-            onKeyDown={(event) => handleKeyDown(event)}
-            onClick={() => {
-              handleTileClick(icons[icon]?.name);
-            }}
-          >
-            <i
-              className={`swirl-icons-${icons[icon]?.name}28 text-icon-strong`}
-            ></i>
-            <IconName />
-          </a>
+          <SwirlPopoverTrigger popover="icon-popover">
```

Comment:
**issue:** looks good now when it comes to opening the tile. But it seems we still have the update issue

https://github.com/getflip/swirl/assets/23700195/f7b9c84c-0f6e-4f37-a0d9-3eca53c7f45a

Reply:
Works fine on my machine. Did you run it locally? If so, you have to run the swirl component builds.

Reply:
I only made a clean install. forgot i had to build it. works!